src/App.tsx

```diff
@@ -1,12 +1,41 @@
-import { BrowserRouter as Router } from "react-router-dom";
-import Navigation from "./Navigation";
-import Page from "./Page";
+import { BrowserRouter as Router } from 'react-router-dom'
+import Navigation from './Components/Navigation/Navigation'
```

Comment:
Please create `index.ts` file which will export all components defined in `Components` folder
```suggestion
import { Navigation } from 'components'
```
Also don't use upper letter for names of folders which are not dedicated for single component (`Navigation` is ok, due to includes `Navigation` file, but `Components` is only folder for many components)